src/Settings.php

```diff
@@ -503,6 +459,57 @@ public function __construct()
             'planet4_settings_features' => Features::get_options_page(),
         ];
 
+        // This option should be visible only if the ElasticPress plugin is activated.
+        $is_elasticpress = function_exists('is_plugin_active') && is_plugin_active('elasticpress/elasticpress.php');
+        if ($is_elasticpress) {
+            $search_content = array(
+                'planet4_settings_search_content' => [
+                    'title' => 'Search content',
+                    'fields' => [
+                        [
+                            'name' => __('Include archived content in search for', 'planet4-master-theme-backend'),
+                            'desc' => __(
+                                '<b>Important:</b> On change of Include archive content setting, Please kindly',
+                                'planet4-master-theme-backend'
+                            ) . ' <a href="admin.php?page=elasticpress-sync" target="_blank">Sync Elasticsearch</a>.',
```

Comment:
nit: we can remove the `target="_blank"` here

Reply:
Good idea, done!